README.md

```diff
@@ -50,7 +50,7 @@ daemon_mode: True
 
 #### Run
 ```
-$ python3 cerberus.py --config <config_file_location>
+$ python3 src/cerberus.py --config <config_file_location>
```

Comment:
@aakarshg this might be confusing as there's no src directory in our repo. Maybe we can change it to:
$ git clone https://github.com/openshift-scale/cerberus.git
$ cd cerberus
$ ## Set the config in config/config.ini
$ python3 cerberus.py --config config/config.ini

Reply:
the PR moved cerberus.py to src/cerberus.py

Reply:
Sorry missed that.